workflows/diagnostics/fv3net/diagnostics/offline/_input_sensitivity.py

```diff
@@ -78,11 +71,20 @@ def _get_variable_indices(
     return variable_indices
 
 
+def _stack_sample_data(ds: xr.Dataset) -> xr.Dataset:
```

Comment:
Moved the stacking out of the input sensitivity calculations so the input here is the same as what the `Predictor.input_sensitivity` method expects. I guess I should have saved this for the follow on PR but I'd already commited this change.